ls1intum/ArTEMiS
# [WIP] Complaints for modeling exercises

### Checklist

### Motivation and Context
We want to offer a complaint feature for assessments that can be used by students if they feel like a submission is not fairly assessed or they want to request more feedback from tutors.

### Description
Re-use the existing complaint mechanism and make it available also for modeling exercises.

### Steps for Testing
1. Log in to ArTEMiS as a student
2. Navigate to assessed modeling submission
3. Click on "Submit complaint or ask for more feedback"
4. Enter and submit a complaint
5. Log in to ArTEMiS as a tutor (not the assessor of the submission from above)
6. Navigate to tutor exercise dashboard for the corresponding exercise
7. See list of complaints and click on "Evaluate the complaint"
8. Reply to complaint
9. Log in as student again
10. Check response for previously submitted complaint
